.github/workflows/ci.yml

```diff
@@ -13,6 +13,9 @@ on:
 jobs:
   test:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: ['3.0', '3.1']
```

Comment:
Rails 6.0 does not support Ruby 3.2 -- and actually only 7.0.X officially supports 3.2 at this point. We can update the matrix when we get onto newer Rails.